build-scripts/build-csdk.js

```diff
@@ -18,11 +18,12 @@ var spawnSync = require( "child_process" ).spawnSync;
 var fs = require( "fs" );
 var repoPaths = require( "./helpers/repo-paths" );
 var shelljs = require( "shelljs" );
+var os = require( "os" );
 
-function run( command, arguments, options ) {
+function run( command, args, options ) {
```

Comment:
Weird! I never got any complaints, but, of course, you're right. Let's keep `args`.